bootstrap/src/main/java/io/airlift/bootstrap/Bootstrap.java

```diff
@@ -233,8 +235,9 @@ public Injector initialize()
             throw new ApplicationConfigurationException(errors, warnings);
         }
 
-        // Log effective configuration
-        if (!quiet) {
+        // Log effective configuration. The config property can force either true/false,
+        // irrespective of if quiet() was set for this Bootstrap instance.
+        if ((!quiet && !Boolean.TRUE.equals(quietConfig)) || Boolean.FALSE.equals(quietConfig)) {
```

Comment:
I think this would be simpler `if (!(quietConfig == null ? quiet : quietConfig)) {`